policyengine_uk/parameters/gov/hmrc/study_childcare_entitlement/care_to_learn_age_eligible.yaml

```diff
@@ -0,0 +1,10 @@
+description: The Department for Education sets a maximum age for carers to be eligible for care to learn scheme.
+metadata:
+  unit: year
+  period: year
+  label: Maximum age to be eligible for care to learn scheme
```

Comment:
Always refer to proper nouns in capitalisation

Reply:
So this should always be Care to Learn- copy however the govt calls schemes. 
<img width="917" alt="image" src="https://github.com/user-attachments/assets/5213ce2c-52d7-43ad-8c7f-d785f926ceec" />